desktop/src/com/mygdx/game/DesktopLauncher.java

```diff
@@ -11,6 +11,7 @@ public static void main (String[] arg) {
 		Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
 		config.setTitle("tda367");
 		config.setWindowedMode(800, 480);
+		config.setWindowIcon("entities/BlueSamurai-Idle-down.png");
```

Comment:
name of file should be changed to something like application-icon or game-icon ect